src/Http/Controllers/Corporation/ViewController.php

```diff
@@ -188,6 +188,70 @@ public function getStarbases($corporation_id)
     }
 
     /**
+<<<<<<< HEAD
+     * @param $corporation_id
+     *
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     */
+    public function getStarbaseSilos($corporation_id)
+    {
+
+        // not pretty but required
+        ini_set('max_execution_time', 12000);
```

Comment:
I have been wondering a bit about this. Should we not rather see if we can optimize the query somehow to not need a random `ini_set` here?